api/src/chat/localization/RcdaTextEnglish.ts

```diff
@@ -37,6 +39,11 @@ export class RcdaMyanmarTextEnglish {
     askTownshipName = "What township are you reporting on?";
     askDisasterType = "What is the disaster type?";
     askGeographicalSettingType = "What is the setting?";
+    tellCurrentAdminStack = (currentAdminStack: string) => `The current admin stack selected is - **${this.townships[currentAdminStack]}**. Do you want to change it?`;
```

Comment:
This is more of a prompt/question than just telling them information. Think we could rename the attribute to something like "askToConfirmAdminStack"?